### DISCUSSION TITLE
Installing db2util on IBM i with yum

### DISCUSSION THREAD

**Discussion by NextPage66** [codefori/vscode-ibmi#1740] [IBM i] [answered]
I am trying to get the source dates in my RPG source using Code for IBM i.  
I read a different discussion about needing db2util on IBM i, but that discussion was closed.
How do I get db2util installed?

**Comment by worksofliam** [comment, marked as answer]
Db2util is not longer required and hasn't been for some time (more than a year for sure). Just enable source dates in the connection settings and you should be good to go.

**Comment by NextPage66** [comment]
Thank you.
